build.gradle

```diff
@@ -704,6 +704,7 @@ project('spring-integration-xmpp') {
 		compile "org.igniterealtime.smack:smack-extensions:$smackVersion"
 
 		testCompile project(":spring-integration-stream")
+		testCompile "org.igniterealtime.smack:smack-experimental:$smackVersion"
```

Comment:
Don't be in a hurry. It is `testCompile`. 
Everything else is up to end-user :smile: